src/prommis/precipitate/precipitator.py

```diff
@@ -10,10 +10,43 @@
 # All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
 # for full copyright and license information.
 #################################################################################
-"""
-Basic precipitator model
-"""
+r"""
+Preliminary Precipitator Unit Model
+===================================
+
+Author: Alejandro Garciadiego
+
+The Precipitator Unit Model represents an Equilibrium reactor unit model with fixed partition coefficients unit model.
+
+Configuration Arguments
+-----------------------
+
+The presipitator unit model needs an aqueos property package which includes stoichiometric values for solids being
+created in the precipitator and fixed separation coefficients of the solids.
+
+Model Structure
+---------------
+
+The Precitator unit model has hard coded stream names (``aqueous`` and ``precipitate`` respectively). The Precipitator 
+model also has one inlet and two outlets named ``aqueous_inlet``, ``aqueous_outlet`` and ``precipitate_outlet`` respectively.
+
+Additional Constraints
+----------------------
 
+The Precipitator unit adds two additional constraint to define the stochiometry and separation.
+
+.. math:: n_{t,prec,c} = /frac{n_{t,aq_in,c} - n_{t,aq_out,c}}{S{comp}} 
+
+where :math:`n_{t,prec,c}` is the outlet precipitacion of c component, :math:`n_{t,aq_in,c}` is the inlet of c comp in 
+the aqueous phase, :math:`n_{t,aq_in,c}` is the outlet of c comp in the aqueous phase at time :math:`t`, divided by the
+stechiometric parameter of component c :math:'{S{comp}'
```

Comment:
```suggestion
stechiometric parameter of component c :math:`S{comp}`
```

Reply:
Also - stoichiometric is spelt incorrectly.

Reply:
I've added it for consideration for the next update to our favorite spellchecker: https://github.com/crate-ci/typos/issues/956#issuecomment-2079532012 (not sure if or when it'll become active,  though, so we'll need to check for it manually until then)